### ISSUE TITLE
[PI] lr-Snes9x - Black Screen in Retropie

### ISSUE THREAD

**Issue by darksaviorx** [libretro/libretro-meta#88]
All games load to a black screen. Snes9x-2010 is fine when built from source. It's been happening for at least a month. Here's the build log: https://pastebin.com/aHeHS08L

**Comment by hizzlekizzle** [comment]
if it's fine when you build from source, there must be something up with how the retropie folks are building it. That's not something we can do anything about, so you might have better luck taking it up with them.

**Comment by darksaviorx** [comment]
I might have confused you. Building lr-snes9x from source within retropie loads all games to a black screen. Building lr-snes9x-2010 from source is fine. I just wanted to make a comparison.

**Comment by hizzlekizzle** [comment]
oh, I see what you mean. Ok, I'll ask around if anyone else can reproduce.

**Comment by hizzlekizzle** [comment]
@darksaviorx 
Nobody I spoke with could reproduce, so there may be something else going on. Can you try one from our buildbot? https://buildbot.libretro.com/nightly/linux/armhf/

**Comment by darksaviorx** [comment]
Thanks for looking into it. I tried snes9x_libretro.so from the link you gave me, but it won't run and gives me an error. I don't use linux enough to understand it. Error is: "relocation error: /opt/retropie/libretrocores/lr-snes9x/snes9x_libretro.so: symbol _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE10_M_replaceEjjPKcj, version GLIBCXX_3.4.21 not defined in file libstdc++.so.6 with link time reference"

I reproduced the problem on my second pi with a completely updated retropie.
